aten/src/ATen/native/cpu/PowKernel.cpp

```diff
@@ -2,7 +2,7 @@
 #include <ATen/Dispatch.h>
 #include <ATen/Parallel.h>
 #include <ATen/cpu/vec256/vec256.h>
-#include <ATen/native/TensorIterator.h>
+#include <ATen/native/TensorIterator.h> 
```

Comment:
Whitespace seems to have crept in, but style tests don't seem to be looking for extra whitespace after `#include` s